.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,308 @@
+name: Canary - Deploy Next Branch
+on:
+  ## Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+  ## Set up a schedule for the workflow to run
+  schedule:
+    - cron: '0 2 * * *' # run at 0200 UTC
+
+jobs:
+  test_core:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
question: shouldn't we check if there are any changes on the `next` branch before running this?

Reply:
It was my understanding to do it once a night versus on each merge with the next branch

We can change that if you guys want though!

Reply:
any update on if you want to do it on push to `next` or just nightly?